src/Utilities/universalfallback.jl

```diff
@@ -939,3 +939,38 @@ end
 MOI.add_variable(uf::UniversalFallback) = MOI.add_variable(uf.model)
 
 MOI.add_variables(uf::UniversalFallback, n) = MOI.add_variables(uf.model, n)
+
+function MOI.get(
+    uf::UniversalFallback,
+    attr::MOI.ListOfVariablesWithAttributeSet,
+)
```

Comment:
The rule is to check `if !MOI.is_copyable(attr) || MOI.supports(uf.model, attr, MOI.VariableIndex)`. If this is true, we query the inner model. Otherwise, we have to handle it with the fallback. Branching on whether it is a key is not correct since we might redirect the call to the inner model for attributes that it doesn't support.

Reply:
Fixed, and added tests